eos-add-flatpak-apps-repos

```diff
@@ -32,4 +32,11 @@ EOF
 # Add the repo only if not added yet
 flatpak remote-add --if-not-exists --from eos-sdk ${TMPDIR}/eos-sdk.flatpakrepo
 
+# Change com.endlessm.EknServices over to the version that uses the new runtime
+# in the eos-sdk repo
+flatpak install eos-sdk com.endlessm.EknServices//eos3
```

Comment:
Doing `flatpak install` at boot time is pretty bad -- we should avoid it at all costs if we can.
Can't you do some surgery and change the origin on the existing one to pull from the new repo instead? Then gnome-software would pull the update normally from there.